web/src/app/(app)/globals.css

```diff
@@ -80,7 +80,9 @@
   /* @apply rounded-lg p-2 overflow-x-scroll */
   @apply  p-2 max-w-full overflow-auto w-full
 }
- 
+.vsc-controller{
+  position: absolute;
+}
```

Comment:
Added to fix a little issue with the videos. The controllers html element is not absolute, which adds a little padding to the left of the video.
The html element is nothing, I think that there is only to capture some events.